bin/test.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * Test runner for Flex Plugin Builder
+ * This is invoked via `npm run test`. You can pass additional parameters using `npm run test -- param1 param2`
+ * To run a single test, use `npm run test:one packages/path/to/file/or/directory
+ */
+
+const fs = require('fs');
+const path = require('path');
+const cp = require('child_process');
+const os = require('os');
+
+// These packages are ignored and no jest runs for them
+const ignorePackages = ['flex-plugin-e2e-tests'];
+
+// fs helper commands
+const isWin = () => os.platform() === 'win32';
+const exists = (src) => src && fs.existsSync(src);
```

Comment:
what is src here? Where is it being set?

Reply:
This is a function, so whoever is calling it is passing the sr